pages.fr/common/borg.md

```diff
@@ -0,0 +1,33 @@
+# borg
+
+> Outil de sauvegarde avec déduplication.
+> Crée des sauvegarde distante ou locale qui peuvent être montée comme un système de fichier.
+> Pour plus d'information: <https://borgbackup.readthedocs.io/en/stable/usage/general.html>.
+
+- Initialise un dépôt local:
+
+`borg init {{/path/to/repo_directory}}`
+
+- Sauvegarde un répertoire dans le dépôt en créant une archive appelée "Monday":
```

Comment:
```suggestion
- Sauvegarde un répertoire dans le dépôt en créant une archive appelée "Lundi":
```
Please also change this in the command accordingly.

Reply:
Ok, I was wondering at translation time if that would be a good idea to change also the command.

Reply:
do you think we should translate also /path/to/repo_directory and so on in various command examples ?

Reply:
@dbrgn yes we should :) see for example some of the Italian translations I made, or the review I left for you [on your PR for `ssh`](https://github.com/tldr-pages/tldr/pull/3157#pullrequestreview-257618521). It would be better to translate tokens *when it makes sense*. For example, if a token is `{{backup}}` and the word "backup" is also used in French that would be ok to not translate.

Reply:
You probably meant @dgsb :)

Reply:
Ouch, sorry about that @dbrgn :sweat_smile: damn GH UI.